### ISSUE TITLE
{ cwd = '~/.config/nvim' } with live_grep not working

### ISSUE THREAD

**Issue by fhill2** [nvim-telescope/telescope.nvim#389]
### Description
When calling `:lua require'telescope.builtin'.live_grep{ cwd = '~/.config/nvim' } `built in live_grep command, specifying a custom path is ignored, and instead always defaults to nvim cwd path

builtin find_files module is working okay when passing a different cwd

everything working okay when using ripgrep outside of nvim, with:
rg --color=never --no-heading --with-filename --line-number --column --smart-case searchstring ~/.config/nvim

I was using a Telescope version around a week old and have recently updated to today's version, and at the same time updated to latest nvim appimage.

**Expected Behavior**
Telescope builtin module live_grep searches through only files located in the path passed in via cwd:
`:lua require'telescope.builtin'.live_grep{ cwd = '~/.config/nvim' }`
OR
`:lua require'telescope.builtin'.live_grep{ cwd = '.config/nvim/' }`
OR
`:lua require'telescope.builtin'.live_grep{ cwd = '/home/userName/.config/nvim/' }`

should search only nvim config files

**Actual Behavior**
Telescope builtin module live_grep search always defaults to nvim current working directory no matter what path is passed in via cwd
The above command always searches files under location of :pwd command output in nvim.

### Details

<details><summary>Reproduce</summary>
1 nvim -nu /path/to/yourdebugtest.vim

2 :lua require'telescope.builtin'.live_grep{ cwd = '~/.config/nvim' }
OR
:lua require'telescope.builtin'.live_grep{ cwd = '.config/nvim/' }
OR
:lua require'telescope.builtin'.live_grep{ cwd = '/home/userName/.config/nvim/' }  

</details>

<details><summary>Environment</summary>
- nvim version: NVIM v0.5.0-dev+1000-g84d08358b **.appimage**
- Operating system: Linux
- Telescope commit: 5d121ee58cf45aa3fd591e6c51054f9fbe38e83a
- ripgrep 12.1.1
- plenary.vim & popup.vim updated to latest version with vim-plug

</details>

<details><summary>Configuration</summary>
<p>
 

```viml
set nocompatible hidden laststatus=2

if !filereadable('/tmp/plug.vim')
  silent !curl --insecure -fLo /tmp/plug.vim
    \ https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
endif

source /tmp/plug.vim
call plug#begin('/tmp/plugged')
Plug 'nvim-lua/popup.nvim'
Plug 'nvim-lua/plenary.nvim'
Plug 'nvim-telescope/telescope.nvim'
call plug#end()

autocmd VimEnter * PlugClean! | PlugUpdate --sync | close
lua << EOF

require('telescope').setup{
  defaults = {
    vimgrep_arguments = {
      'rg',
      '--color=never',
      '--no-heading',
      '--with-filename',
      '--line-number',
      '--column',
      '--smart-case'
    },
    prompt_position = "bottom", 
    prompt_prefix = ">",
    selection_strategy = "reset",
    sorting_strategy = "descending",
    layout_strategy = "horizontal",
    layout_defaults = {},
    file_ignore_patterns = {},
    shorten_path = true,
    winblend = 0,
    width = 0.75,
    preview_cutoff = 120,
    results_height = 1,
    results_width = 0.8,
    border = {},
    borderchars = { '─', '│', '─', '│', '╭', '╮', '╯', '╰'},
    color_devicons = true,
    use_less = true,
  }
}
EOF
```

**Comment by Conni2461** [comment]
Thanks for the report. Will be fixed in #390

**Comment by fhill2** [comment]
I can confirm this is fixed. Can't reproduce.
